CMakeLists.txt

```diff
@@ -1782,8 +1782,40 @@ target_sources(${PROJECT_NAME} PRIVATE
         core/gdxsv/gdxsv_patch.inc
         core/gdxsv/gdxsv_emu_hooks.cpp
         core/gdxsv/gdxsv_emu_hooks.h
+		core/gdxsv/gdxsv_translation.h
+		core/gdxsv/gdxsv_translation.cpp
+		core/gdxsv/gdxsv_translation_patch_utf8.h
+		core/gdxsv/gdxsv_CustomTexture.h
+		core/gdxsv/gdxsv_CustomTexture.cpp
         core/gdxsv/lbs_message.h
         core/gdxsv/mcs_message.h)
+
+if(APPLE)
+		set(ASSETS core/gdxsv/Textures)
+		target_sources(${PROJECT_NAME} PRIVATE ${ASSETS})
+		source_group("Resources" FILES ${ASSETS})
+		set_source_files_properties(${ASSETS} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
+elseif(WIN32)
+		add_custom_target(gdxsv_translation_generate_texture ALL
+			COMMAND bash "./gen_textures_rc.sh"
+			WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/core/gdxsv/"
+			VERBATIM
+		)
+		add_dependencies(${PROJECT_NAME} gdxsv_translation_generate_texture)
+		set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/core/gdxsv/textures.rc PROPERTIES GENERATED 1)
+		target_sources(${PROJECT_NAME} PRIVATE core/gdxsv/textures.rc)
+endif()
+
+if(APPLE OR WIN32)
+	add_custom_target(gdxsv_translation_patch ALL
+		COMMAND ${CMAKE_COMMAND} -E echo "//  Shift_JIS generated by iconv. DO NOT EDIT" > gdxsv_translation_patch.h
+			COMMAND iconv -f UTF-8 -t SHIFT-JIS gdxsv_translation_patch_utf8.h >> gdxsv_translation_patch.h 
+			WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/core/gdxsv/"
+			VERBATIM
```

Comment:
I would like to avoid subprocess execution in cmake if possible.
Let's create a gdxsv_translation_patch directory in root directory like `gdxsv_path` and generate the header `gdxsv_tlanslation_path.h`

Reply:
Sure, using subprocess execution is the fastest way for me to quickly patch the strings and build a binary back and forth, and I think the heaviest task is completed now and no more rapid changing is required.

Reply:
Fixed.